pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -156,6 +156,13 @@ in
     nvimRequireCheck = "advanced_git_search.utils";
   };
 
+  # TODO: optional dependency but required for toggleterm
```

Comment:
I'm aware of that but this is an interesting case. This is a vim plugin that had optional lua modules so we might need to add a doCheck skip because there is no main lua module

Reply:
Oh, looked at this plugin a bit closer. It is more of a Vim plugin than Neovim, so we can safely disable nvimRequireCheck and maybe enable vimCommandCheck (or something similar? I don't remember the name of that hook)